1208-projects/my_projects.feature

```diff
@@ -1,8 +1,23 @@
 Feature: User My Projects
+    Background: A User is logged in
```

Comment:
Er `Background:` tenkt som et cucumber step eller er det bare en del av beskrivelsen?

Reply:
Eg tolker det til å vere ein felles forutsettning for alle scenario i denne feature, jamfør:
"A Background allows you to add some context to the scenarios that follow it. It can contain one or more Given steps, which are run before each scenario, but after any Before hooks.

A Background is placed before the first Scenario/Example, at the same level of indentation."
https://cucumber.io/docs/gherkin/reference/#background